### DISCUSSION TITLE
How to add an ant path animation to a line

### DISCUSSION THREAD

**Discussion by fabripetrelli** [uber/react-map-gl#2143] [Q&A] [answered]
Hi everyone, <br/>I would like to recreate [this example](https://docs.mapbox.com/mapbox-gl-js/example/animate-ant-path/) made in mapbox-gl-js docs but there's not `setPaintProperty` in the ref that i pass to the `ReactMapGL` component.

How can I achieve this result?

**Comment by Pessimistress** [comment, marked as answer]
Just update the `paint` prop of your `<Layer>`.
